core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -39,8 +39,10 @@ import scala.reflect.ClassTag
 object RequestChannel extends Logging {
   private val requestLogger = Logger("kafka.request.logger")
 
-  val RequestQueueSizeMetric = "RequestQueueSize"
-  val ResponseQueueSizeMetric = "ResponseQueueSize"
+  val DataPlaneRequestQueueSizeMetric = "DataPlaneRequestQueueSize"
+  val DataPlaneResponseQueueSizeMetric = "DataPlaneResponseQueueSize"
```

Comment:
Yes. But I was thinking if we can change the names to explicitly specify that these metrics are meant for data plane. Let me know if that is Ok. Since this is a server side change, the clients should be unaffected. If we do this, the Kafka service providers will have to change there monitoring infra, in case they have alerts set on this. If you think this is OK, I will change the KIP to point it out. If we think, this is not needed, I will revert this change.

Reply:
@MayureshGharat : Since many people may be depending on the existing metric for monitoring, perhaps it's better to keep the existing metric name.